scalafix-rules/src/main/scala/scalafix/internal/rule/ExplicitResultTypes.scala

```diff
@@ -18,11 +18,14 @@ final class ExplicitResultTypes(
 ) extends SemanticRule("ExplicitResultTypes") {
 
   def this() = this(ExplicitResultTypesConfig.default, LazyValue.now(None))
+  val supportedScalaVersions = BuildInfo.supportedScalaVersionforRules
+  val runtimeScalaVersion = BuildInfo.scalaVersion
+
+  private def toBinaryVersion(v: String) = v.split('.').take(2).mkString(".")
 
-  private def supportedScalaVersion = Properties.versionNumberString
   override def description: String =
     "Inserts type annotations for inferred public members. " +
-      s"Only compatible with Scala $supportedScalaVersion."
+      s"Only compatible with Scala $supportedScalaVersions."
```

Comment:
does the `toString` representation of that map reads well? maybe we should only show the values (and maybe mkstring them) ?
```suggestion
      s"Only compatible with Scala ${supportedScalaVersions.values.flatten}."
```